### PR TITLE
feat: added footer component

### PR DESCRIPTION
**Pull request by Kblob1**
## Describe your changes

Created a footer component that can go at the bottom of any page on the website. Can change the content of paragraphs later.

Here is an Image without the page being filled:
Screenshot (161)

Made sure that it fills to the bottom when items are added. Like the Alabama Caregiver Connect Website.

### Checklist

- [ ] Unit tests added or not applicable
- [ ] Does not introduce any breaking changes
- [ ] Ran `Prettier`

### CODE REVIEW

File: client/src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        //     <div id="footer-text">
+        //         <p><a href="https://accessibility.ua.edu/statement/">Accessibility</a> | <a href="https://eop.ua.edu/">Equal
+        //             Opportunity</a> | <a href="https://www.ua.edu/disclaimer">Site Disclaimer</a> | <a
+        //             href="https://www.ua.edu/privacy">Privacy</a></p>
+        //         <p><a href="https://www.ua.edu/copyright">Copyright © 2022 </a> | <a href="https://www.ua.edu/">The
+        //             University of Alabama</a> | Tuscaloosa, AL 35487 | (205) 348-6010</p>
+        //         <p>Website provided by the <a href="https://cit.ua.edu/">Center for Instructional Technology</a>, <a
+        //             href="https://oit.ua.edu/">Office of Information Technology</a></p>
+        //     </div>
+        // </div>
```
Comment (kevinleey) [inline client/src/components/Footer.js:19]: Can you delete this commented out code?

**Review by kevinleey — COMMENTED** [review]
Changes look good, would you mind adding a screenshot of the footer into the PR desc? Thanks.

File: client/src/components/Footer.js (hunk 1 above)
Comment (Kblob1) [reply client/src/components/Footer.js]: Got it! Thank you for the catch.

**Review by kevinleey — APPROVED** [review]
We'll need to add some arbitrary page space so the footer is pushed all the way to the bottom but this looks great :shipit:

**Pull request merged**